src/styles/webfonts/rubik/index.css

```
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: "Rubik";
+  src: url("./rubik-light.ttf") format("ttf");
```

According to MDN there isn't a `ttf` type. Rubik was applied only after I changed it to `truetype`, otherwise it was always using `sans-serif` :thinking: 

```suggestion
  src: url("./rubik-light.ttf") format("truetype");
```